src/ethereum_test_specs/state_static.py

```diff
@@ -0,0 +1,223 @@
+"""Ethereum General State Test filler static test spec parser."""
+
+from functools import cached_property
+from typing import Callable, ClassVar, Dict, List
+
+import pytest
+from pydantic import Field, field_validator
+
+from cli.fillerconvert.structures.common import AddressInFiller
+from cli.fillerconvert.structures.expect_section import (
+    AccountInExpectSection,
+)
+from cli.fillerconvert.structures.state_test_filler import StateTestInFiller, StateTestVector
+from ethereum_test_base_types import Address, Bytes, Hash, HexNumber, Storage, ZeroPaddedHexNumber
+from ethereum_test_forks import Fork, get_forks
+from ethereum_test_types import Account, Alloc, Environment, Transaction
+
+from .base_static import BaseStaticTest
+from .state import StateTestFiller
+
+
+class StateStaticTest(BaseStaticTest):
+    """General State Test static filler from ethereum/tests."""
+
+    tests: Dict[str, StateTestInFiller] | None = Field(None)
+    vectors: List[StateTestVector] | None = Field(None)
+    # format_name: ClassVar[str] | None = "state_test"
```

Comment:
```suggestion
    format_name: ClassVar[str] = "state_test"
```
This might have been an issue.